lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -7130,6 +7134,27 @@ private long importCcFromStream(InputStream fis) {
 		    bos.close();
 
 		    CartridgeLoader cartridgeLoader = ZipLoader.getUtilities(cc, root.getCanonicalPath());
+
+		    // Force the unzip to happen as it would have happened anyways
+			if ( importArchive ) {
```

Comment:
spacing off

Reply:
The spacing in the lines above and below is a combination of tabs and spaces.  I am not going to match that in new code :)